demos/src/inline.mustache

```diff
@@ -0,0 +1,165 @@
+    <div class="o-forms__inline-container">
+        <div class="o-forms__inline-item">
+            <legend class="o-forms__label">Default radio controls</legend>
+            <div class="o-forms__additional-info">Prompt text follows the ledgend if required.</div>
+        </div>
+        <div class="o-forms__group">
+            <input type="radio" name="radio" value="1" class="o-forms__radio" id="radio11" checked="checked" />
+            <label for="radio11" class="o-forms__label">Radio 1</label>
+            <input type="radio" name="radio" value="2" class="o-forms__radio" id="radio12" />
+            <label for="radio12" class="o-forms__label">Radio 2</label>
+        </div>
+    </div>
+</fieldset>
+
+<fieldset class="o-forms o-forms--inline o-forms--radios o-forms--error">
+    <div class="o-forms__inline-container">
+        <legend class="o-forms__label">Error</legend>
+        <div class="o-forms__group">
+            <input type="radio" name="radio3" value="1" class="o-forms__radio" id="radio31" checked="checked" />
+            <label for="radio31" class="o-forms__label">Radio 1</label>
+            <input type="radio" name="radio3" value="2" class="o-forms__radio" id="radio32" />
+            <label for="radio32" class="o-forms__label">Radio 2</label>
+        </div>
+        <div class="o-forms__errortext">Invalid entry</div>
+    </div>
+</fieldset>
+
+<div class="o-forms o-forms--inline">
+    <div class="o-forms__inline-item">
+	    <label for="select-standard" class="o-forms__label">Select box</label>
+	    <div class="o-forms__additional-info">Prompt text follows the label if required.</div>
+    </div>
+	<select id="select-standard" class="o-forms__select">
+		<option value="option1" selected>Option 1</option>
+		<option value="option2">Option 2</option>
+		<option value="option3">Option 3</option>
+		<option value="option4">Option 4</option>
+	</select>
+</div>
+
+<div class="o-forms o-forms--inline o-forms--error">
+	<label for="select-invalid" class="o-forms__label">Select box (error)</label>
+    <select id="select-invalid" class="o-forms__select">
+        <option value="option1" selected>Option 1</option>
+        <option value="option2">Option 2</option>
+        <option value="option3">Option 3</option>
+        <option value="option4">Option 4</option>
+    </select>
+    <div class="o-forms__errortext">Invalid entry</div>
+</div>
+
+<div class="o-forms o-forms--inline">
+    <div class="o-forms__inline-item">
+	    <label for="o-forms-standard" class="o-forms__label">Text input</label>
+	    <div class="o-forms__additional-info">Prompt text follows the label if required..</div>
+    </div>
+	<input type="text" id="o-forms-standard" class="o-forms__text" required />
+</div>
+
+<div class="o-forms o-forms--inline o-forms--error">
+	<label for="o-forms-invalid" class="o-forms__label">URL input :invalid</label>
+    <input type="url" id="o-forms-invalid" class="o-forms__text" value="hp://invalid.com" />
```

Comment:
I found that aria-describedby on the input, referencing the error message, will cause it to be read when the input is focussed.

Reply:
Ahh this is very nice. I'll update all the demos.

Reply:
It works the same for you? 
Try it in My account - enable controlCentrePreferencesPage flag then hit ft.com/preferences, edit Personal details, blank the First name field and hit enter

Reply:
Have you got the same approach to work with fieldset/legend?

Reply:
I'm not totally sure I understand your question. It works great for me using VoiceOver on My Account, and the technique works for us in our demos where the error is displayed.

Reply:
👍

Reply:
I ended up adding aria-describedby to the demo for additional content but not errors. But I've noted it on a separate issue for errors. 👍